src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj

```diff
@@ -0,0 +1,89 @@
+<Project>
+
+  <PropertyGroup>
+    <OutputInRepoRoot>true</OutputInRepoRoot>
+  </PropertyGroup>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  <Import Project="..\Microsoft.AspNetCore.App.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion)</TargetFramework>
```

Comment:
Should we define this at some lower level (root Directory.props maybe)? Seems like something that would be very commonly used across the repo.

Reply:
Probably. Updated to add a property named DefaultNetCoreTargetFramework. This could be used in a lot of places, but for now I'll just update the only two places currently using `netcoreapp$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion)`